src/amo/server/base.js

```diff
@@ -404,10 +404,10 @@ function baseServer(
           _log.debug(oneLine`${req.method} -> ${responseStatusCode},
             redirecting=${isRedirecting} anonymous=${!token}:
             response should be cached.`);
-          res.append('Cache-Control', 's-maxage=180');
+          res.append('Cache-Control', 's-maxage=360');
           // The following is for backwards-compatibility until we have
           // switched nginx to obey Cache-Control instead.
-          res.set('X-Accel-Expires', '180');
+          res.set('X-Accel-Expires', '360');
```

Comment:
Have we switched nginx to obey Cache-Control instead?

Reply:
We have done that a long time ago _and_ switched away from nginx caching for the frontend but I didn't want to remove that yet to avoid changing too many things at once. I'll file a follow-up cleanup issue.